CBinding/CMake/CMakeProject.cs

```diff
@@ -43,7 +43,7 @@ public class CMakeProject : FolderBasedProject
 		string name;
 		FilePath outputDirectory = new FilePath ("./bin");
 		CMakeFileFormat fileFormat;
-
+		GeneralOptionsPanel guiOptions = new GeneralOptionsPanel ();
```

Comment:
It's bad practice to use UI code from non-UI code. It runs the risk of introducing coupling issues and threading issues.

Instead use `PropertyService` to read the settings stored by the panel.